examples/worlds/triggered_publisher.sdf

```diff
@@ -387,13 +387,17 @@ start falling.
         <time>0.001</time>
         <enabled>true</enabled>
       </plugin>
-      <plugin filename="ignition-gazebo-detachable-joint-system" name="ignition::gazebo::systems::DetachableJoint">
+      <plugin
+        filename="ignition-gazebo-detachable-joint-system"
+        name="ignition::gazebo::systems::DetachableJoint">
```

Comment:
Just as a matter of course, it is best if you don't add unrelated changes to the PR.  It makes it easier to review, and keeps each PR focused.  The same goes several times below in the XML, and in the code itself (where there are instances of adding in extra brackets and such).

Reply:
Yeah I noticed that myself. I saw that some of the original code had lines of code that exceeded 80 chars or didn't have the angled brackets. But from next time, I won't touch the unrelated changes to the PR so that it's not as complicated for the reviewer

Reply:
Oh I asked Liam to fix those, because they were not adhering to style. We do these "by the way" small fixes in Gazebo sometimes, because otherwise they never get fixed. It's too small for a PR of its own.